CHANGELOG.md

```diff
@@ -28,6 +28,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   Firecracker snapshots (up to Firecracker version v1.6.0) incompatible with
   the current Firecracker version.
 
+### Deprecated
+
+- Firecracker's `--start-time-cpu-us` parameter is deprecated and will be
+  removed in v2.0. It is used by the jailer to pass the value that should be
+  subtracted from the CPU time, but in practice it is always 0. The parameter
+  was never meant to be used by end customers, but we recommend doing any such
```

Comment:
nit: the `but` is a little bit confusing. Is it supposed to urge customers to take an action that they haven't been taking so far?

Reply:
yes, sounds good